tests/e2e/README.md

```diff
@@ -158,6 +159,16 @@ For example:
 - `PUPPETEER_SLOWMO=10` - will run tests faster
 - `PUPPETEER_SLOWMO=70` - will run tests slower
 
+### How to run tests in debug mode
```

Comment:
Maybe we could include a note or a link to a page explaining how debug mode can be used to debug the tests? Asking as I don't know and I'm sure it is super helpful.

Reply:
@rodrigoprimo Thanks for the suggestion. I created #27975.